src/VisualStudio/Core/Def/HACK_ThemeColorFixer.cs

```diff
@@ -16,8 +21,10 @@ namespace Microsoft.VisualStudio.LanguageServices
     /// from its default blue. As a work around, we listen to <see cref="IClassificationFormatMap.ClassificationFormatMappingChanged"/>
     /// and update the classification format maps that we care about.
     /// </summary>
-    [Export]
-    internal sealed class HACK_ThemeColorFixer
+    [Export(typeof(IWpfTextViewConnectionListener))]
+    [ContentType(ContentTypeNames.RoslynContentType)]
+    [TextViewRole(PredefinedTextViewRoles.Interactive)]
```

Comment:
The docs [here](https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.text.editor.iwpftextviewconnectionlistener.aspx) seem to imply that _some_ TextViewRole must be specified

Maybe Analyzable would be better? I'm not really sure. @olegtk, any advice?